src-electron/db/query-session.js

```diff
@@ -40,6 +40,65 @@ async function getAllSessions(db) {
   return rows.map(dbMapping.map.session)
 }
 
+/**
+ * Reloads a previous session.
+ *
+ * @export
+ * @param {*} db
+ * @param {*} sessionId
+ * @param {*} userRef
+ * @param {*} sessionKey
+ * @returns A promise that resolves with the number of rows updated.
```

Comment:
This does not seem right. Is more than one row actually updated or just one row?